tests/regr_test.py

```diff
@@ -103,67 +170,71 @@ def test_testcase_directory(package: PackageInfo, version: str, platform: str, q
         "--no-error-summary",
         "--platform",
         platform,
-        "--no-site-packages",
         "--strict",
         "--pretty",
+        "--no-incremental",
```

Comment:
I'm not sure whether this is necessary or not. In theory it might make the various mypy runs more independent. But we haven't had an actual problem with the mypy cache so far, so this might just make mypy slower unnecessarily.

@hauntsaninja, do you have any opinions here?

Reply:
Unless we have a concrete reason, I think no need for `--no-incremental`

Reply:
```suggestion
```